views/templates/admin/configuration.tpl

```diff
@@ -55,7 +55,7 @@
 		{l s='Errors list:' mod='mailjet'}
 		<ul class="mj_errors">
 			{foreach from=$MJ_errors item=current_error}
-				<li>{$current_error|escape|default:''}</li>
+				<li>{$current_error|default:''}</li>
```

Comment:
This is not working - the link we must display is broken when we use "{...|escape:'javascript'}" either "{...|escape:'url'}" or any other variation of "escape"